css/parser/tests/fixture/function/url/span.rust-debug

```diff
@@ -0,0 +1,437 @@
+  |                       ^^^^^^^^^^^^^^^^^^^^^^
+
+error: Text
+ --> $DIR/tests/fixture/function/url/input.css:5:23
+  |
+5 |     background-image: URL('./image (1).jpg');
+  |                       ^^^
+
+error: Value
+ --> $DIR/tests/fixture/function/url/input.css:5:27
+  |
+5 |     background-image: URL('./image (1).jpg');
+  |                           ^^^^^^^^^^^^^^^^^
+
+error: Str
+ --> $DIR/tests/fixture/function/url/input.css:5:27
+  |
+5 |     background-image: URL('./image (1).jpg');
+  |                           ^^^^^^^^^^^^^^^^^
+
+error: Property
+ --> $DIR/tests/fixture/function/url/input.css:6:5
+  |
+6 |     background-image: url(   './image (1).jpg');
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: Text
+ --> $DIR/tests/fixture/function/url/input.css:6:5
+  |
+6 |     background-image: url(   './image (1).jpg');
+  |     ^^^^^^^^^^^^^^^^
+
+error: Value
+ --> $DIR/tests/fixture/function/url/input.css:6:23
+  |
+6 |     background-image: url(   './image (1).jpg');
+  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: FnValue
+ --> $DIR/tests/fixture/function/url/input.css:6:23
+  |
+6 |     background-image: url(   './image (1).jpg');
+  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: Text
+ --> $DIR/tests/fixture/function/url/input.css:6:23
+  |
+6 |     background-image: url(   './image (1).jpg');
+  |                       ^^^
+
+error: Value
+ --> $DIR/tests/fixture/function/url/input.css:6:30
+  |
+6 |     background-image: url(   './image (1).jpg');
+  |                              ^^^^^^^^^^^^^^^^^
+
+error: Str
+ --> $DIR/tests/fixture/function/url/input.css:6:30
+  |
+6 |     background-image: url(   './image (1).jpg');
+  |                              ^^^^^^^^^^^^^^^^^
+
+error: Property
+ --> $DIR/tests/fixture/function/url/input.css:7:5
+  |
+7 |     background-image: url('./image (1).jpg'   );
+  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: Text
+ --> $DIR/tests/fixture/function/url/input.css:7:5
+  |
+7 |     background-image: url('./image (1).jpg'   );
+  |     ^^^^^^^^^^^^^^^^
+
+error: Value
+ --> $DIR/tests/fixture/function/url/input.css:7:23
+  |
+7 |     background-image: url('./image (1).jpg'   );
+  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: FnValue
+ --> $DIR/tests/fixture/function/url/input.css:7:23
+  |
+7 |     background-image: url('./image (1).jpg'   );
+  |                       ^^^^^^^^^^^^^^^^^^^^^^^^^
+
+error: Text
+ --> $DIR/tests/fixture/function/url/input.css:7:23
+  |
+7 |     background-image: url('./image (1).jpg'   );
+  |                       ^^^
+
+error: Value
+ --> $DIR/tests/fixture/function/url/input.css:7:27
+  |
+7 |     background-image: url('./image (1).jpg'   );
+  |                           ^^^^^^^^^^^^^^^^^^^^
+
+error: SpaceValues
+ --> $DIR/tests/fixture/function/url/input.css:7:27
+  |
+7 |     background-image: url('./image (1).jpg'   );
+  |                           ^^^^^^^^^^^^^^^^^^^^
```

Comment:
Is this intentional?
The property value should be [`UrlValue`](https://docs.rs/swc_css_ast/0.5.0/swc_css_ast/struct.UrlValue.html), not `SpaceValues`.